aws_quota/cli.py

```diff
@@ -192,6 +203,13 @@ def check(check_keys, region, profile, warning_threshold, error_threshold, fail_
                     checks.append(
                         chk(session, identifier)
                     )
+            elif issubclass(chk, RegionQuotaCheck):
+                try:
+                    for rg in check_regions(regions):
```

Comment:
nit: Since this is reused in a bunch of places you could replace it with a function on the RegionQuotaCheck class that creates multiple checks when given a region.